service.subtitles.legendasdivx/resources/settings.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
just a note, this is the old settings layout. Kodi is migrating to have the same style of settings accross addons and the core: https://kodi.wiki/view/Add-on_settings_conversion

Not mandatory and we still don't know if the old settings format will be kept for v19 or only be dropped in v20

Reply:
ok